### DISCUSSION TITLE
Phrasing of Github About Text is missing "is"

### DISCUSSION THREAD

**Discussion by ghost** [bromite/bromite#2199] [General]
Currently says "Bromite a Chromium fork with ad blocking and privacy enhancements; take back your browser!"
I assume you meant "Bromite is a Chromium fork with ad blocking and privacy enhancements; take back your browser!"

I tried creating an issue but it didn't fit any of the templates, so I thought I'd post here

https://github.com/bromite/bromite/issues/2198

**Comment by csagan5** [comment]
I have fixed it, thanks.

The template mentions that for anything which is not a bug or a feature request, discussions should be used.

**Reply by ghost** [reply]
ah ok
